SentinelHub/tests/test_utils_geo.py

```diff
@@ -0,0 +1,69 @@
+# coding=utf-8
+from typing import Tuple, TypeVar
+
+import pytest
+
+pytest.importorskip("qgis.core")
+
+from qgis.core import QgsRectangle  # noqa: E402
+
+from ..settings import Settings  # noqa: E402
+from ..utils.geo import bbox_to_string, get_custom_bbox, is_bbox_too_large, is_supported_crs  # noqa: E402
+
+T = TypeVar("T", float, int)
+
+
+@pytest.mark.parametrize(
+    "lat_max, lat_min, lng_max, lng_min, coords",
```

Comment:
what if you have just `input_coords, expected_coords` and then do
```python
lat_max, lat_min, lng_max, lng_min = input_coords
```
at the start of the test